content/docs/command-reference/checkenv.md

```diff
@@ -0,0 +1,11 @@
+# checkenv
+
+Check that current environment satisfies object requrements
+
+## Options
+
+- `-p, --project TEXT`: Path to MLEM project [default: (none)]
```

Comment:
style: `-p <path>`, `--project <path>` - path to MLEM ... 

default - check how it's done in the docs

TEXT should be renamed into path, also in the CLI itself

Reply:
done, but sometimes the duplication of metavar feels strange